### PR TITLE
WMS ID # 3120 : Migrate WebLogic Server to OCI with Autonomous Database updates

### PR DESCRIPTION
**Pull request by nujehowlett**
@streamnsight Outstanding updates identified by OHC Learn Publishing Team

### CODE REVIEW

**Review by tmcginn — COMMENTED** [review]
@streamnsight - are you ok with these changes?

File: developer-library/weblogic-to-oci/migrate-app-db-atp-datapump/migrate-app-db-atp-datapump.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ There are 2 ways to download the wallet on to the target WebLogic servers:
     </copy>
     ```
 
-    *Make sure to specify the proper domain name (here `nonjrf_domain`)* if you didn't follow the naming convention of this workshop.
+    Make sure to specify the proper domain name (here `nonjrf_domain`) if you didn't follow the naming convention of this workshop.
```
Comment (streamnsight) [inline developer-library/weblogic-to-oci/migrate-app-db-atp-datapump/migrate-app-db-atp-datapump.md:367]: I would keep the emphasis here as it is important.

**Review by streamnsight — CHANGES_REQUESTED** [review]
@nujehowlett 
A few minor comments for your review. 
Thanks

File: developer-library/weblogic-to-oci/intro-oci/intro-oci.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ In this lab, you will:
 
 ### Prerequisites
 
-*In order to run this workshop you need:*
+In order to run this workshop you need:
 
 * A Mac OS X, Windows or Linux machine.
-* A private/public SSH key-pair.
+* A private or public SSH key-pair.
```
[the remark this answers is not in the document]
Comment (nujehowlett) [reply developer-library/weblogic-to-oci/intro-oci/intro-oci.md]: @streamnsight I have updated as requested.

File: developer-library/weblogic-to-oci/migrate-app-db-atp-datapump/migrate-app-db-atp-datapump.md (hunk 1 above)
Comment (nujehowlett) [reply developer-library/weblogic-to-oci/migrate-app-db-atp-datapump/migrate-app-db-atp-datapump.md]: @streamnsight I have updated as requested.

**Review by tmcginn — APPROVED** [review]

**Pull request merged**